cmd/habitat/community/manager.go

```diff
@@ -67,16 +70,13 @@ func (m *Manager) checkCommunityExists(communityID string) bool {
 
 }
 
-func (m *Manager) CreateCommunity(name string, id string, createIpfs bool) (*community.Community, error) {
+func (m *Manager) CreateCommunity(name string, createIpfs bool) (*community.Community, error) {
```

Comment:
i think we want to still pass `id` in. For the case where we want to just create the raft files for a community id that this node wants to join.

Reply:
We shouldn't call `CreateCommunity` when a member is joining. All common stuff between them should be covered by the internal raft service